tools/ARIAtools/extractProduct.py

```diff
@@ -49,17 +49,20 @@ def createParser():
             required=True, help='ARIA file')
     parser.add_argument('-w', '--workdir', dest='workdir', default='./',
         help='Specify directory to deposit all outputs. Default is local directory where script is launched.')
-    parser.add_argument('-gp', '--gacos_products', dest='gacos_products', type=str, default=None,
-        help='Path to director(ies) or tar file(s) containing GACOS products. Will use new version of products (.tif) if they exist.'\
-             'Further information on GACOS available at: http://www.gacos.net.')
+    parser.add_argument('-gp', '--gacos_products', dest='gacos_products',
+                        type=str, default=None, help='Path to director(ies) '
+                        'or tar file(s) containing GACOS products.')
+    parser.add_argument('-tc', '--tropo_corrections', dest='tropo_corrections',
```

Comment:
Im not sure i am yet in favor of this approach. It would make more sense to add "troposphere" or "troposphereTotal" to the list of extracted variables. i see the use of TC perhaps in future for different models but we are not there yet. We should discus more.

Reply:
Agreed, I removed it now and cleaned up the logic